README.md

```diff
@@ -0,0 +1,18 @@
+# Filters
+
+## Usage with ament_cmake
+
+Here is recommended approach on how to link `filters` to your project, using the `filter_base` target.
```

Comment:
One small request: could you add one more sentence explaining to users what depending on filters_base does? Something along the lines of "linking to filters_base pulls in all necessary libraries and include directories for targets with classes that extend FilterBase".

Reply:
And also a comment for the `add_library(filter_base ...)` in the CMakelists.txt for old ROS folks like me who are coming across interface libraries for the first time. Something like "filter_base is a cmake "interface library" which doesn't compile anything, but can be depended on by other targets so that all needed libraries and include directories are pulled in"

Reply:
Done and done. I reworked it it a bit. Most people probably want to link to one of the pre-written filters.

Reply:
I like the way you worded the comments.